### PR TITLE
Docs format fix

### PR DESCRIPTION
**Pull request by yf225**
Docs demo: http://parl.ai/static/docs/agents.html

### CODE REVIEW

File: parlai/core/agents.py [hunk 1]
```diff
@@ -333,3 +233,109 @@ def report(self):
...
+    try:
+        # Tries to call the create_agent function in agents.py
+        create_agent = getattr(my_module, 'create_agents')
+        task_agents = create_agent(opt)
+    except AttributeError:
+        # Create_agent not found, so try to create the teacher directly.
+        return create_task_agent_from_taskname(opt)
+    if type(task_agents) != list:
+        task_agents = [task_agents]
+    return task_agents
```
Comment (alexholdenmiller) [inline parlai/core/agents.py]: this code is all the same, right?

**Comment by alexholdenmiller** [comment]
awesome!! thank you!

**Comment by alexholdenmiller** [comment]
merging so I can do work on the task_list file

File: parlai/core/agents.py (hunk 1 above)
Comment (yf225) [inline parlai/core/agents.py]: Yes it is - moving them around a little bit so that in the doc all of the classes will show first and then the methods :)

**Pull request merged**